### PR TITLE
Modify to use utils

### PR DESCRIPTION
**Pull request by gongweibao**

### CODE REVIEW

File: go/filemanager/pfsserver/handler.go [hunk 1]
```diff
@@ -20,64 +19,17 @@ type response struct {
...
-	defer resp.Body.Close()
-
-	if resp.StatusCode != http.StatusOK {
-		return []byte{}, errors.New("server error: " + resp.Status)
-	}
-	// FIXME: add more resp.Status checks.
-	return ioutil.ReadAll(resp.Body)
-}
-
 var TokenUri = "http://cloud.paddlepaddle.org"
```
Comment (Yancey1989) [inline go/filemanager/pfsserver/handler.go]: Here can not be a hard code.

**Comment by Yancey1989** [comment]
I think we should a doc that explains how to deploy the pfsserver on Kubernetes, I created a issue: https://github.com/PaddlePaddle/cloud/issues/138

File: go/filemanager/pfsmodules/cp.go [hunk 2]
```diff
@@ -67,3 +70,80 @@ func (p *CpCmd) PartToString(src, dst string) string {
...
+	f.BoolVar(&p.V, "v", false, "Cause cp to be verbose, showing files after they are copied.")
+}
+
+// Execute runs CpCmd.
+func (p *CpCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}) subcommands.ExitStatus {
+	if f.NArg() < 2 {
+		f.Usage()
+		return subcommands.ExitFailure
+	}
+
```
Comment (typhoonzero) [inline go/filemanager/pfsmodules/cp.go]: Remove these unnecessary blank lines to shorten the file.

Comment (gongweibao) [inline go/filemanager/pfsmodules/cp.go]: 谢谢!

**Pull request merged**